ballerina/Module.md

```diff
@@ -92,26 +92,30 @@ public function main() returns error? {
 }
 ```
 
-2. Use `bal run` command to compile and run the Ballerina program.
+### Step 4: Run the Ballerina application
+
+```bash
+bal run
+```
```

Comment:
in the other connector doc this instruction is mentioned under step 3. Shall we stick with the same convention? WDYT?

Reply:
In the `Gmail` connector at https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail, the same convention is followed.

Reply:
Addressed with https://github.com/ballerina-platform/module-ballerinax-twilio/pull/230/commits/080154505136df9d98ef24c063e9afb82223c670